src/system/data/actor/common.ts

```diff
@@ -3,6 +3,8 @@ import {
     Resource,
     AttributeGroup,
     Skill,
+    Currency,
+    CurrencyDenomination,
     ExpertiseType,
```

Comment:
Looks like there are still references to `Currency` and `CurrencyDenomination` even though they are no longer defined.